staging/src/k8s.io/apiserver/pkg/endpoints/discovery/aggregated/handler.go

```diff
@@ -144,8 +150,10 @@ type priorityInfo struct {
 
 func NewResourceManager(path string) ResourceManager {
 	scheme := runtime.NewScheme()
+	utilruntime.Must(apidiscoveryv2scheme.AddToScheme(scheme))
+	utilruntime.Must(apidiscoveryv2beta1scheme.AddToScheme(scheme))
```

Comment:
Ah this is the spot where it's constructed.  Here is the point where you would add `RegisterConversions(scheme)`.  You may even be able to move the conversions into this package and make them private `registerConversions`.

Reply:
ack, updated